### ISSUE TITLE
georeferenceVerificationStatus - Darwin Core Hour Input Form 2/14/2017 10:35:02

### ISSUE THREAD

**Issue by iDigBioBot** [tdwg/dwc-qa#32]
A user submitted this information via the Darwin Core Hour [webform](https://docs.google.com/forms/d/e/1FAIpQLSekd0QSdgiK5ujh6CLZPx9kKTcS-mJRNK_pGWk3b3fHv4x4gQ/viewform): 
*Timestamp:* 2/14/2017 10:35:02
*Please provide a topic of interest:* Do you use the georeferenceVerificationStatus field? Do you know if or how researchers have used this field?
*Are you capable of and interested in participating:* Yes
*Who else would you recommend to participate in the presentation:* John Wieczorek
*What resources can you point to:* http://arctos.database.museum/info/ctDocumentation.cfm?table=ctverificationstatus
*Your name:* Erica Clites
*Your email:* eclites@berkeley.edu

**Comment by tucotuco** [comment]
This answer for this submission has been [documented](https://github.com/tdwg/dwc-qa/wiki/Georeferences#georeferenceVerificationStatus) directly.

**Comment by eclites** [comment]
Follow-up question. Why only the three recommended terms for this field? Wouldn't it be helpful to researchers to know that georeferenced points have had some checking, but hasn't gone to the level of "verified by curator"? For example, the "checked by curator" field used in the Arctos db. http://arctos.database.museum/info/ctDocumentation.cfm?table=ctverificationstatus

Also wondering about changing the georeferencedBy field after verification. If I don't change the point generated, it seems counterproductive to lose who originally did this work?

**Comment by tucotuco** [comment]
I would argue that those additional Arctos values are useful in-house for data management and georeferncing projects, but not particularly useful when data are shared. "Not horribly wrong" is something that needs to be verified, after all, regardless of who said so.

I see what you mean about georeferencedBy. Again, this might be a difference in the management perspective versus the sharing one. In the latter, your users want to know if they should trust the data. That might be easier to know if the one responsible for the data is the one who is verifying it and putting their stamp of approval on the georeference - essentially stating that it can't be done better than it is with available information.

**Comment by eclites** [comment]
Thanks! This distinction between what's useful for day to day management of your collection and what's useful to aggregators is an important one I hadn't quite grasped before.

**Comment by eclites** [comment]
Since georeferencedBy can be concatenated list of people, maybe all of the appropriate people can be listed here. Is there a preference for order in order to indicate the approval you mentioned?

**Comment by tucotuco** [comment]
This is the first time the subject has come up for me. To me it seems
natural to put the most authoritative responsible party first on the list.
Presumably that would normally also be the most recent one. If that seems
reasonable, then it would suggest reverse chronological order. And for
those done jointly, maybe have them in alphabetical order.

**Comment by debpaul** [comment]
Hi @tucotuco @eclites. Note that it has come up quite a few times in workshops - that people sometimes see a need to keep more than one georeference (and the associated "who dunnit" fields). It could be in the future @eclites, if you had such a situation, you could use an extension to show that you have more than one georeference for a given object. I can think of one example (for which @tucotuco may have a different idea) where the locality written description and the lat/lon, both provided by the collector, do not line up. IOW, if you georeference from the written description, your point ends up in a place different from the lat/lon (that was provided by the collector). What to do? In a perfect world, I think, w/o checking with the collector, we'd share both georeferences making it clear that the provenance of one, is from the collector, the other is a georeference by (whomever) based on the written description.

**Comment by eclites** [comment]
Thanks, @debpaul! When you say use an extension, do you mean a DwC extension? There is the verbatimLat/verbatimLong and verbatimCoordinates fields as well. If there were only two points you needed to serve, you could serve one via verbatim and the other via decimalLat/decimalLong.

**Comment by debpaul** [comment]
Hi @eclites Hm. We might need a new ticket (we're going in a few directions here). Yes, i mean a dwc extension. I wouldn't suggest using the verbatim / decimalLat/Lon fields quite that way :-) They are meant to show a relationship between coordinates that might be in a non decimal degrees format and the resulting derived decimal degrees coordinates. So the TRS, or degrees-minutes-seconds, or UTM northing and easting can go in verbatimCoordinates, but the derived (converted) decimal degrees go into the respective decimalLat and decimalLon fields.

**Comment by eclites** [comment]
@debpaul Got it, my mistake!

**Comment by debpaul** [comment]
No worries @eclites. You raise great points. It's always the case that standards, software, hardware, the data itself, and humans, are evolving to meet needs as they are understood and can be addressed. We sometimes find out we can't quite do what we'd like with our data - and then we work on making it possible :-) as you are doing.

**Comment by tucotuco** [comment]
Documented at https://github.com/tdwg/dwc-qa/wiki/Georeferences#georeferenceVerificationStatus.

**Comment by DMNHverts** [comment]
How about cases where the georeferencing has been done by a third party (such as for the ORNIS project)?  Verified by curator seems not quite right.  And verified by collector and requires verification don't really work.

**Comment by tucotuco** [comment]
I would argue that these georeferences remain unverified until someone accepts them as the best that can be done with the information known to exist. If that assessment has not been done, it still requires verification.

**Comment by DMNHverts** [comment]
Thanks!  I wasn’t sure how to submit a comment on a closed thread so I’m glad you found it.  Your solution sounds good-  I’m prepared to stand behind the ones I did but I know the ORNIS georeferencing was done in bulk and didn’t have the advantage of field notes and other information to help find some places.  Even since 2008 the quality and quantity of maps available online has changed dramatically.

Enjoyed your webinar the other day although I had to join late.  I clearly need to re-read your opus on georeferencing and see what else I’m missing.

Thanks very much- Jean

Jean L. Woods, Ph.D.                                                    Phone: 302-658-9111 x314
Curator of Birds                                                             Fax: 302-658-2610
Delaware Museum of Natural History                      jwoods@delmnh.org
P.O. Box 3937                                                                www.delmnh.org
(4840 Kennett Pike)
Wilmington, DE  19807

Feast on the Beach: The Delaware Bay Horseshoe Crab Shorebird Connection<http://www.delmnh.org/feast-on-the-beach/> – a new film from the Delaware Shorebird Project
